go/webpack/cbor_format_test.go

```diff
@@ -31,10 +31,14 @@ func TestWriteCBOR(t *testing.T) {
 	pack := Package{
 		parts: []*PackPart{
 			&PackPart{
-				url:            staticUrl("https://example.com/index.html?query"),
-				requestHeaders: HTTPHeaders{},
-				status:         200,
+				requestHeaders: HTTPHeaders{
+					httpHeader(":method", "GET"),
```

Comment:
Is it all right if I submit this PR and send a separate one to remove `:method`? I think that'll do better to focus discussion on whether it's a good idea to diverge from HTTP2 here.